petab/core.py

```diff
@@ -521,9 +521,12 @@ def handle_missing_overrides(mapping_par_opt_to_par_sim, observable_ids):
         for i_condition, mapping_for_condition in \
                 enumerate(mapping_par_opt_to_par_sim):
             for i_val, val in enumerate(mapping_for_condition):
-                if isinstance(val, numbers.Number):
+                try:
```

Comment:
If observable ids starting with ``(noise|observableParameter[0-9]+_ can`` safely be assumed to be override parameters, then one can also remove the observable_id iteration

Reply:
Let's assume that.